src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -311,12 +302,21 @@ namespace ts.projectSystem {
         checkNumberOfInferredProjects(projectService, count.inferredProjects || 0);
     }
 
+    export function configuredProjectAt(projectService: server.ProjectService, index: number) {

```

Comment:
sure, but for now I will leave this. Because we were earlier using this as configuredProjects[index] and it was easier to replace those with this call than to go look at all the tests. I will create issue to track this.